### PR TITLE
add main-container

### PR DESCRIPTION
**Pull request by IanWen103026**
add main-container to application.html.erb

edit index.html.erb & application.html.erb & custom.css

propose: avoid main-container from appearing repeatedly in multiple html.erb

### CODE REVIEW

**Comment by AaronWu-train** [comment]
Please describe the changes and propose in the PR message

**Comment by IanWen103026** [comment]
avoid main-container from appearing repeatedly in multiple html.erb

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,21 +1,72 @@
...
-<%= yield %>
-  </body>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+  <script src="jquery-sakura.js"></script>
+  <script>
+    $(window).load(function () {
+        $('.content').sakura();
+    });
+  </script>
+  <script>
```
Comment (ckeisc43rd-cjtsai) [inline app/views/layouts/application.html.erb]: explicit this to external javascript file

**Review by ckeisc43rd-cjtsai — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -12,6 +12,11 @@
                            'data-turbo-track': 'reload',
                            type: 'module'
   %>
+  <%=
+  javascript_include_tag 'main-container',
+                         'data-turbo-track': 'reload',
+                         type: 'module'
+%>
```
Comment (ckeisc43rd-cjtsai) [inline app/views/layouts/application.html.erb]: format html file with prettier

**Review by ckeisc43rd-cjtsai — CHANGES_REQUESTED** [review]

**Review by ckeisc43rd-cjtsai — DISMISSED** [review]

**Comment by ckeisc43rd-cjtsai** [comment]
LGTM

**Comment by AaronWu-train** [comment]
@IanWen103026 @ckeisc43rd-cjtsai Check the solved conflict
is it correct?

**Review by AaronWu-train — COMMENTED** [review]
@IanWen103026 Reviewed. All conflicts are solved.

**Comment by IanWen103026** [comment]
Correct.

**Review by AaronWu-train — APPROVED** [review]
OK

**Pull request merged**